cloudant-client/src/main/java/com/cloudant/client/api/model/DesignDocument.java

```diff
@@ -152,8 +155,9 @@ public JsonObject getIndexes() {
      * Get the changes feed filter functions defined in this design document.
      *
      * @return map of filter name to function string
-     * @see <a target="_blank"
-     * href="https://docs.cloudant.com/design_documents.html#filter-functions">Filter functions</a>
+     * @see <a
+     * href="https://console.bluemix.net/docs/services/Cloudant/api/design_documents.html#filter-functionss"
```

Comment:
Extra `s` needs deleting from end of link

Reply:
Fixed in 7a6c69c.